libs/application-generic/src/usecases/workflow/update-workflow/update-workflow.command.ts

```diff
@@ -16,7 +16,7 @@ import {
 } from '@novu/shared';
 
 import { EnvironmentWithUserCommand } from '../../../commands';
-import { NotificationStep } from '../../../usecases';
+import { NotificationStep } from '../..';
```

Comment:
```suggestion
import { NotificationStep } from '../../usecases';
```

I often find these implicit imports from the index file to be troublesome for introduction of cyclic dependencies. Can we revert?